src/risedevtool/src/task/compute_node_service.rs

```diff
@@ -67,6 +67,13 @@ impl ComputeNodeService {
             .arg("--streaming-memory-proportion")
             .arg(&config.streaming_memory_proportion.to_string());
 
+        if config.disable_streaming {
+            cmd.arg("--disable-streaming");
+        }
+        if config.disable_serving {
+            cmd.arg("--disable-serving");
+        }
```

Comment:
To enhance clarity and comprehension, I suggest replacing 'disable' with 'enable' in all configurations to ensure straightforwardness.